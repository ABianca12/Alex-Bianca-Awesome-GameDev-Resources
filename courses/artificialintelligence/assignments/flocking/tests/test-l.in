1.000 0.000 0.000 0.000 1.000 0.000 0.000 3
0.000 0.500 0.000 0.000
0.000 0.000 1.100 0.000
0.000 -0.500 0.000 0.000
0.010
0.010
