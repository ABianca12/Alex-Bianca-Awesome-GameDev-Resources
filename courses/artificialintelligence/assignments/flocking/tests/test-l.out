0.000 0.492 0.000 -0.750
0.011 0.000 1.100 0.000
0.000 -0.492 0.000 0.750
0.000 0.478 0.006 -1.489
0.022 0.000 1.089 0.000
0.000 -0.478 0.006 1.489
